requirements/dev.txt

```diff
@@ -5,4 +5,7 @@ nox==2020.8.22
 pre-commit>=2.9.2
 isort>=5.6.4
 types-PyYAML==5.4.3
-types-mock>=4.0.8
\ No newline at end of file
+types-mock>=4.0.8
+kornia>=0.6.4
+opencv-python>=3.4
+hydra-core>=1.1.2
```

Comment:
We should avoid adding example specific requirements to the dev reqs. We could add to the examples/README.md (like some of the others) or we could add another reqs file for example related things?

Also speaking of readme, we should update that too.